packages/live-share-media/src/MediaPlayerSynchronizer.ts

```diff
@@ -87,6 +87,7 @@ export class MediaPlayerSynchronizer extends EventEmitter {
     private _onPlayerEvent: EventListener;
     private _seekSuspension?: MediaSessionCoordinatorSuspension;
     private _viewOnly = false;
+    private _blockNonExplicitPlayerEvents = true;
```

Comment:
name suggestion: can we simply call `_blockNonPlayerEvents` instead?

Reply:
@pradeepananth when true, it will block player events unless they went through the synchronizer. It has a notion of `_expectedPlaybackState`, and only player events that align with that expected state are not blocked. So that label wouldn't really be accurate.

Reply:
We could change it to be `_blockNonExpectedPlayerEvents`, which might align better. But it needs some sort of word like expected or explicit to be accurate.

Reply:
or `_blockUnexpectedPlayerEvents`?

Reply:
@huntj88 I like that